@here/harp-map-controls/lib/Utils.ts

```diff
@@ -86,3 +86,35 @@ export function getWidthAndHeightFromCanvas(
         height: safeParseDecimalInt(canvas.style.height, canvas.clientHeight)
     };
 }
+
+export interface CameraPoseOptions {
+    geoCoordinate: GeoCoordinates;
+    distance?: number;
+    azimuth?: number;
+    altitude?: number;
+    name?: string;
+}
+/**
+ * Defines a Camera Pose, meaning its target, direction and distance.
+ */
+export class CameraPose {
```

Comment:
please add units, i.e. degrees or radians for `azimuth`, `altitude` and `distance`

Reply:
Removed the CameraPose completly as it basically duplicates the LookAtParams from MapView